<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balanceDialog</class>
 <widget class="QDialog" name="balanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Takaisin</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="title">
    <string>Saldo</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="lblBalance">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>21</pointsize>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string>0000 €</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblCreditlimit">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Luottoraja: 0000 €</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>541</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Viimeisimmät tapahtumat</string>
   </property>
   <widget class="QTableWidget" name="tblHistory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Nimi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Aika</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nosto</string>
     </property>
    </column>
    <item row="0" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="2">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
